### ISSUE TITLE
EMU 0204 distorted sound (wrong sample rate?)

### ISSUE THREAD

**Issue by marcsabat** [KXStudio/Repository#189]
Hello,

Since the last week or so sound is distorted when using my DAC (EMU 0204) on my Ubuntu 18.04 + kstudio repos. I know the DAC is a bit old but was working fine up until a week ago or so (and it continues to do so under a vanilla Ubuntu 18.04).

Could this be caused by some changes in alsa?

When using alsamixer there's an option to change the samplerate. That usually solved this problem in the past but not anymore: although it works (I mean it changes the samplerate) there's not a single setting that sound correct.

I tried both with regular audio (from web browser and clementine) and also through jack (Ardour). In both cases I get the same distorted result. I assume that when enabling Jack Pulseaudio is stoped so I narrow the problem to a deeper level (alsa?).

I wanted to uninstall kxstudio repos and packages to know for sure if kxsutdio packages are the culprit but I don't know how to do it: if I disable kxstudio repositories alsa packages stay the same (I don't know how to revert them back to vanilla Ubuntu packages).

**Comment by falkTX** [comment]
I recommend to try a live-cd of Ubuntu, and then AVLinux and see how is the behaviour there.

**Comment by marcsabat** [comment]
Both with Ubuntu and AVLinux live cd's work as expected.
Does AVLinux has the exact same alsa drivers as kxstudio repos for Ubuntu?
If it's so then I guess that the problem might be only on my side.

**Comment by marcsabat** [comment]
I'll close the issue (I am not sure if kxstudio packages have something to do or not yet).

In case it may be of any use for some other people here are my findings so far:

When playing something through **EMU-0204 it does not work if sample rate is 44100Hz**. Works fine if it's 48000khz.

If I set default-sample-rate=48000 to /etc/pulse/daemon.conf the system seems to work.
However:
1. I didn't need to do this before.
2. When setting up an Ardour project to use 44100hz (using Jack) the same behaviour appears and I had been recording with those settings not so long ago (same hardware).

So, to sum up: something is messed up and setting the device to 44100hz does not work properly (distorted sound).
I tried with a different PC (with stock Ubuntu and the same device): 44100hz audio works fine.

*In order to check the configured sample rate I use this command:
cat /proc/asound/card1/pcm0p/sub0/hw_params

Any clue will be appreciated. I still don't discard a change in alsa drivers is the cause of this.
